pallets/manta-pay/src/runtime.rs

```diff
@@ -16,12 +16,12 @@
 
 //! MantaPay Runtime APIs
 
-use crate::{DensePullResponse, PullResponse, RawCheckpoint};
+use crate::{PullResponse, RawCheckpoint, RuntimeDensePullResponse};
 
 sp_api::decl_runtime_apis! {
     pub trait PullLedgerDiffApi {
         fn pull_ledger_diff(checkpoint: RawCheckpoint, max_receivers: u64, max_senders: u64) -> PullResponse;
```

Comment:
you could probably unify these to also return RuntimeDensePullResponse here and just convert to a `PullResponse` in the client, gets rid of a type and you could get rid of `dense_pull_ledger_diff` as a runtime API altogether. (Client API will then introduce the difference depending on the string encoding)

Reply:
I think about removing `dense_pull_ledger_diff`, not sure it will make people confused if without a dedicated runtime api. And some one might call runtime api by `state_call`.

Reply:
Nvm, seems we're the only user, I think we should keep one runtime api `pull_ledger_diff`.

Reply:
yes. people interfacing with the RT directly can also do the re-serialization themselves